src/iccpd/src/iccp_cli.c

```diff
@@ -117,7 +117,17 @@ int set_peer_link(int mid, const char* ifname)
                        csm->mlag_id, ifname);
     }
 
+    if (MAX_L_PORT_NAME > strlen(csm->peer_itf_name))
```

Comment:
is this check correct?  I thought the protection is to ensure that the name is not greater than what is the allowed max value?  but the way it is checking it seems to be the other way around?  can you please check again this logic?